hw7/ex36.py

```diff
@@ -0,0 +1,8 @@
+def print_operation_table(operation, num_rows=6, num_columns=6):
+    for x in range(1, num_rows + 1):
+        lists = []
+        for y in range(1, num_columns + 1):
+            num = operation(x, y)
+            lists.append(num)
+        print(*[str(x) for x in lists])
```

Comment:
Почему бы не преобразовать в строковый тип в момент добавления в список? Тогда не придется делать list comprehention в 7 строке.
```suggestion
            lists.append((str(num))
        print(*lists)
```